ansible/roles/vault_utils/tasks/push_secrets.yaml

```diff
@@ -63,6 +63,7 @@
     found_file: "{{ lookup('ansible.builtin.first_found', findme) }}"
   vars:
     findme:
+      - "~/.patterns/values-secret-{{ pattern_name }}.yaml"
       - "~/values-secret-{{ pattern_name }}.yaml"
       - "~/values-secret.yaml"
       - "{{ pattern_dir }}/values-secret.yaml.template"
```

Comment:
At some point I would like to see this list pruned back to:
~/.patterns/values-secret-{{ pattern_name }}.yaml
{{ pattern_dir }}/values-secret.yaml.template
and VALUES_SECRET

Reply:
agree